apps/extension/src/ui/apps/onboard/routes/Password.tsx

```diff
@@ -103,9 +101,9 @@ export const PasswordPage = () => {
         name: "Submit",
         action: "Choose password continue button",
       })
-      navigate(isResettingWallet ? "/account" : `/privacy`)
+      navigateNext()
```

Comment:
need to revert this line change, as navigateNext is memoised, this will run with isLoggedIn === FALSE

otherwise we're redirected to dashboard right after setting password, skipping privacy and account creation screens

Reply:
In this case it wasn't because of memoisation, but simply because before the password is set, `isLoggedIn` is FALSE 🤦‍♂️.